templates/master/00-master/gcp/files/etc-kubernetes-manifests-gcp-routes-controller.yaml

```diff
@@ -0,0 +1,35 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/kubernetes/manifests/gcp-routes-controller.yaml"
+contents:
+  inline: |
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: gcp-routes-controller
+      namespace: kube-system
+    spec:
+      containers:
+      - name: gcp-routes-controller
+        image: "{{.Images.gcpRoutesController}}"
+        command: ["gcp-routes-controller"]
```

Comment:
won't this have to be ["usr-bin-gcp-routes-controller"] like here:

https://github.com/openshift/machine-config-operator/blob/8367a767919c112142f1559d56b0d08aa7bf6613/templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml#L17

Reply:
`/usr/bin` is in `$PATH`, I guess `gcp-routes-controller` is there as well

Reply:
ah fair enough then. ty @runcom